python/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+cmake_minimum_required(VERSION 3.20.1 FATAL_ERROR)
+
+# TODO: Pull this directly from the file written by rapids_cmake_write_version_file.
+set(rmm_version 22.04.00)
```

Comment:
We should use `update-version.sh` to keep the version string in check.

What you want is:

```cmake
project(
  rmm-python
  VERSION 22.04.00
  ...
  )
  
You will also need to update the  `update-version.sh` to modify the `VERSION` entry in this file 
```